test/test.xunit.runner.dnx/project.json

```diff
@@ -1,13 +1,13 @@
 ﻿{
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
```

Comment:
Can we do both here as well? Would be good to know we're not regressing netstandardapp...